scripts/flash/dispute-contribute.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var getTime = require("./get-timestamp");
+var getPrivateKeyFromString = require("../dp/lib/get-private-key").getPrivateKeyFromString;
+var repFaucet = require("../rep-faucet");
+var BigNumber = require("bignumber.js");
+var displayTime = require("./display-time");
+var setTimestamp = require("./set-timestamp");
+
+var day = 108000; // day
+
+function disputeContributeInternal(augur, marketID, outcome, amount, disputerAuth, invalid, auth, callback) {
+  repFaucet(augur, disputerAuth, function (err) {
+    if (err) {
+      console.log(chalk.red("Error"), chalk.red(err));
+      callback(err);
+    }
+    if (err) return console.error(err);
+    invalid = !invalid ? false : true;
+    augur.markets.getMarketsInfo({ marketIDs: [marketID] }, function (err, marketsInfo) {
+      var market = marketsInfo[0];
+      var marketPayload = { tx: { to: marketID } };
+      augur.api.Market.getFeeWindow(marketPayload, function (err, feeWindowId) {
+        var feeWindowPayload = { tx: { to: feeWindowId } };
+        augur.api.FeeWindow.getStartTime(feeWindowPayload, function (err, feeWindowStartTime) {
+          getTime(augur, auth, function (timeResult) {
+            var setTime = parseInt(feeWindowStartTime, 10) + day;
+            displayTime("Current timestamp", timeResult.timestamp);
+            displayTime("Fee Window end time", feeWindowStartTime);
+            displayTime("Set Time to", setTime);
+            setTimestamp(augur, setTime, timeResult.timeAddress, auth, function () {
+              var numTicks = market.numTicks;
+              var payoutNumerators = Array(market.numOutcomes).fill(0);
+              payoutNumerators[outcome] = numTicks;
+              var bnAmount = new BigNumber(amount, 10).toFixed();
+              console.log(chalk.yellow("sending amount REP"), chalk.yellow(bnAmount));
+              augur.api.FeeWindow.isActive(feeWindowPayload, function (err, result) {
+                console.log(chalk.green.dim("Few Window is active"), chalk.green(result));
+                if (result) {
+                  augur.api.Market.contribute({
+                    meta: disputerAuth,
+                    tx: { to: marketID  },
+                    _payoutNumerators: payoutNumerators,
+                    _invalid: invalid,
+                    _amount: bnAmount,
```

Comment:
Numerical arguments to the autogenerated api (`augur.api.*`) should always be provided as 0x-prefixed hex strings; `bnAmount` is a base-10 string.  (Regular JS numbers will work too, but I suggest avoiding these for various reasons.)  Suggest using `"0x" + new BigNumber(amount, 10).toString(16)` on line 38 to obtain a hex string.